src/DefaultEtagGenerator.php

```diff
@@ -0,0 +1,27 @@
+<?php
+/**
+ * Created by PhpStorm.
+ * User: adri
+ * Date: 06/01/16
+ * Time: 16:04
+ */
+
+namespace ZF\HttpCache;
+
+use Zend\Http\Request as HttpRequest;
+use Zend\Http\Response as HttpResponse;
+
+class DefaultEtagGenerator implements EtagGeneratorInterface
```

Comment:
This is one place where we have an exception to our naming rules; we typically use the casing for the acronym, so this would become `DefaultETagGenerator` and `ETagGeneratorInterface`.